WORKING_GROUPS.md

```diff
@@ -0,0 +1,173 @@
+
+Responsibilities include:
+
+- Design Importers and Exporters for different data sets
+- Maintain Unixfs
+- Improve the APIs that let users interact with Files on IPFS
+
+### Runtime Support and Build
+
+The Runtime Support and Build Working Group ensures that the implementations of IPFS are supported by multiple runtimes and architectures, while providing with the necessary set up to empower developers to make the best decisions while contributing to the project.
+
+Responsibilities include:
+
+- Create a Build pipeline for IPFS.
+- Create and monitor continuous testing for the multiple supported platforms (Architecture, OS'es, Browsrs).
+- Explore new runtimes and create Roadmaps towards supporting those.
+
+### QA, Testing and Dev Team Enablement
+
+The QA, Testing and Dev Team Enablement Working Group focuses on developing tools, CI and automating maintenance tasks to improve the Developer experience of the many IPFS contributors.
+
+Responsibilities include:
+
+- Building and upgrading CI for IPFS
+- Automating tasks for things such as: release cycle, ensuring code quality, activity reports.
+- Upgrade dev tools (i.e AEgir)
+
+### Security
+
+The Security Working Group ensures that Security issues are handled with maximum urgency and that risk is mitigated.
+
+Responsibilities include:
+
+- Regular audits by independent parties
+- Create a bugbounty program
+- Create a Responsible disclosure program
+- Create advisories and CVE for any reported vulnerabilities
+- Ensure patches are done
+- Communicate to the community on how to migrate and how to mitigate security risks
+
+### Performance and Benchmarking
+
+The Performance and Benchmarking Working Group is dedicated to optimizing the performance of IPFS.
+
+Responsibilities include:
+
+- Create solid Benchmarks for IPFS and its subsystems
+- Help protocol implementors make the best decisions when it comes to design
+
+### Community, Events & Evangelism
+
+The Community, Events & Evangelism Working Group is responsible for building a environment that fosters collaboration, innovation and inclusiveness for the IPFS community.
+
+Responsibilities include:
+
+- Organize Meetups in multiple cities
+- Help community members organize meetups for their own communities
+- Respond to requests for IPFS presence in conferences
+- Managing the official Social Media channels
+- Publish regular updates to the community (blog posts and newsletters)
+
+### Documentation, Education and Websites
+
+The Documentation, Education and Websites Working Group creates the Documentation and Education materials that help more users and contributors get on board to the IPFS project.
+
+Responsibilities include:
+
+- Write and make sure that docs are up to date.
+- Create Tutorials and Workshops to teach concepts about IPFS and the Distributed Web.
+- Develop the front facing Webpages of each project (ipfs.io, libp2p.io, multiformats.io and ipld.io)
+
```

Comment:
- ensure that public forums (discuss.ipfs.io, irc, github, etc) have adequate moderation and responsive support
- ensure that the code of conduct is followed and provide appropriate support or response when violations occur